router/router.go

```diff
@@ -734,7 +736,19 @@ func (self *Router) initializeHealthChecks() (gosundheit.Health, error) {
 
 func (self *Router) RegisterXweb(x xweb.Instance) error {
 	if err := self.config.Configure(x); err != nil {
-		return err
+		actualErr := false
+
+		for je := err; je != nil; je = errors.Unwrap(err) {
```

Comment:
If you use the `errors` package. I'm pretty sure it handles this unwrapping for you.

Reply:
it did. i must have had the syntax wrong the first time i tried. i was "forced" to use the std erros but clearly it was just a  me problem. i treid it again and it was fine. happy to undo that! :) thx